ruby-challenges.rb

```diff
@@ -30,3 +65,21 @@
 # Expected output example: my_bike.pedal_faster(18) => 28
 # Expected output example: my_bike.brake(5) => 23
 # Expected output example: my_bike.brake(25) => 0
+
+class Bike
+    def initialize(speed)
+      @speed = speed
+    end
+  
+    def pedal_faster(speed_increase)
+      @speed += speed_increase
+      @speed = 0 if @speed < 0
```

Comment:
I am curious what is the current output you are getting. May need to refactor this a bit but you are on the right path.